p4-16/spec/P4-16-spec.mdk

```diff
@@ -4390,6 +4410,13 @@ some unspecified value will be used for that field.
   called, which was not assigned a value during the execution of that
   `control` or `action` (this list of examples is not intended to be
   exhaustive).
+- reading a field of a header that is an element of a header stack,
+  where the index is out of range for the header stack.
+
+TBD: Should an expression like
+`my_hdr_stack[run_time_variable_index].isValid()` be defined as
```

Comment:
Undefined sounds fine. That allows implementations that return 'false'.

Reply:
I have updated the PR to specify that the valid bit of an out-of-range element of a header stack is undefined, with a couple of sentences indicating what that means.